blackbox/docs/release_notes/1.2.x.txt

```diff
@@ -0,0 +1,134 @@
+.. _1.2.x_series:
+
+====================
+1.2.x Release Series
+====================
+
+Versions
+========
+
+.. toctree::
+    :maxdepth: 1
+
+    1.2.0
+
+Upgrade Guide
+=============
+
+.. NOTE::
+
+   To upgrade to the the 1.2.x release series, you must first be running the
+   latest version of the :ref:`1.1.x_series`.
+
+Daemon User
+-----------
+
+You can no longer run CrateDB as the superuser on Unix-like systems. You should
+create a new ``crate`` user for running the CrateDB daemon.
+
+
+Logging
+-------
+
+Your logging config file must be renamed::
+
+    sh$ mv logging.yml log4j2.properties
+
+System Properties
+-----------------
+
+If you have previously set or configured ``CRATE_JAVA_OPTS`` in your startup
+scripts or environment, you should remove this, as CrateDB can no longer be
+configured via this method.
+
+Configuration Changes
```

Comment:
the ticket for this work had a subtask that said "rename changed and remove deprecated settings (also in custom init script if official packages are not used)" can someone pls verify that this section and the one previous properly address that